lib/ansible/modules/network/meraki/meraki_network.py

```diff
@@ -135,6 +141,11 @@
         returned: success
         type: bool
         sample: true
+      disableRemoteStatusPage:
```

Comment:
We return values in snail-casen so this would be: disable_remote_status_page

I personally hate negative variables, I think it's easier to have a `remote_status_page_enabled: no` rather than `disable_remote_status_page: yes`.

Reply:
Should I be manipulating the response from Meraki? For example, if they return `disableRemoteStatusPage: false`, should I rewrite that key and value to be `enable_remote_status_page: yes`? This isn't something I've done for other modules. At a minimum, it makes sense to return enableRemoteStatusPage to be consistent with the other modules. I'm fine changing the output to snail-case but that would break existing playbooks. This one I can change since it's a new value. Thoughts?

Reply:
More on this. Changing from camel to snake will break playbooks so it's a more complicated effort. I created https://github.com/ansible/ansible/pull/53891 to address this.